repose-aggregator/src/docs/asciidoc/recipes/rate-limiting.adoc

```diff
@@ -1,72 +1,124 @@
 = Rate Limiting
 
-include::../_includes/in-progress.adoc[]
+== Overview
+A common use case for *Repose* is rate limiting.
+It limits how many requests per some unit of time (e.g. 10 requests per minute) are allowed to be made.
+In our example, we'll be using the `X-PP-User` header to indicate who we are for rate limiting purposes.
```

Comment:
I might change this line to:
> In this recipe, we will be using the `X-PP-User` header to identity the user for rate limiting purposes.

Subtle change, but I think it helps. That could just be me, though.

Reply:
Sure. That keeps it consistent with the wording.